js-exercises/user-check-password/user.js

```diff
@@ -0,0 +1,19 @@
+
+// (*)
+// Follow the instructions and fill in the blank sections.
+function User(username, password) {
+  // set a username and password property on the user object that is created\
+  this.username = username
+  this.password = password
+}
+
+// create a method on User called `checkPassword`
+// this method should take in a string and compare it to the object's password property
+// return `true` if they match, otherwise return `false`
+User.prototype.checkPassword = function checkPassword(passwordTyped) {
+	return this.password == passwordTyped ? true : false 
```

Comment:
No need of returning true or false explicitly. The expression this.password == passwordTyped already returns boolean. So, we can simply return that boolean.